Sources/XCTest/Public/Asynchronous/XCTWaiter.swift

```diff
@@ -217,9 +217,17 @@ open class XCTWaiter {
         self.manager = manager
 
         // Begin the core wait loop.
+#if os(Windows)
+        let timeoutTimestamp = NSDate._currentAbsoluteTime() + timeout
```

Comment:
Isnt `Date.timeIntervalSinceReferenceDate` the same as `CFAbsoluteTimeGetCurrent()` so I dont think there is a need for this SPI or the Windows/NonWindows differences, it can just be:

`let timeoutTimestamp = Date.timeIntervalSinceReferenceDate + timeout`